src/cryptoadvance/specter/templates/includes/tx-table.html

```diff
@@ -200,6 +200,7 @@
             width: 100%;
             margin-bottom: 10px;
             border: 1.5px solid var(--cmap-border);
+            overflow-x: scroll;
```

Comment:
I think overflow-x: auto would be better choice, considering it handles both overflowing and non overflowing cases. Just to avoid a scroll bar in case of no-overflow.

Reply:
Done

Reply:
Sorry for responding so late: I would have chosen a different approach. I would have added the class `optional` to the time column, similarly to what we have done with the txid column (or to another column that seems not so necessary for limited space on a phone). We ca finetune that later in a different PR if you like the idea.

Reply:
@relativisticelectron Would look like this, let me know what you think, I can quickly make a PR out of it:

![grafik](https://user-images.githubusercontent.com/70536101/186663550-28d6d78d-94db-4f4b-a427-34f2331ee199.png)

Reply:
That would have worked too :-)